docs/cli.md

```diff
@@ -918,27 +922,9 @@ tar -czvf dist/archive.tar.gz <PATH-TO-EXECUTABLE>
 
 {{% /expand%}}
 
-<!-- { {% /tab %}} -->
```

Comment:
I think the motivation was explained [here](https://github.com/viamrobotics/docs/pull/2669#discussion_r1544614872): it was supposed to be supported but not supported yet. I assumed someone would let us know if it was implemented (from the scope doc i had for ticket) but it might be good to check with michael lee as to if there are still any plans to or if this method was scrapped

Reply:
I see - that's annoying.

Reply:
I'll spin this out into its own PR so we can merge the rest